pages/hub/lists/index.tsx

```diff
@@ -66,6 +116,59 @@ const ListsHub: WithPageLayout = () => {
           goToPage
         />
       </div>
+
+      <Dialog open={isDeleteOpen}>
+        <DialogContent className="px-2 max-w-[90%] lg:max-w-xl rounded-t-lg pt-2">
+          <DialogTitle>
+            <Title level={3}>Delete List</Title>
+          </DialogTitle>
+
+          <Text>
+            Are you sure you want to delete <span className="font-bold text-light-slate-12">{listNameToDelete}</span>?
+          </Text>
+          <Text>
+            If you have data on this list that your team is using it would be difficult for your team to get access to
+            track your project.
```

Comment:
I think the copy here could be more clear. Maybe something simple like?

```suggestion
            If you have data on this list that your team is using, they will loose access
```